src/application-base/ApplicationBase.module.scss

```diff
@@ -1,9 +0,0 @@
-:local {
-  .application-base {
-    position: relative;
-  }
-
-  .application-base.fill {
-    height: 100%;
```

Comment:
100% would get applied in the session provider now? or the application container?

Reply:
The ApplicationContainer would be the first structural component to render to the DOM and define that height. I'd expect a SessionProvider to be either rendered within the ApplicationContainer, or to render and ApplicationContainer itself. Haven't decided yet.